pykrige/core.py

```diff
@@ -54,6 +58,26 @@
 from scipy.optimize import minimize
 
 
+def great_circle_distance(lon1, lat1, lon2, lat2):
+    # Calculate dot product of both vectors:
+    lat1 *= np.pi/180.0
+    lat2 *= np.pi/180.0
+    d = np.cos((lon1-lon2)*np.pi/180.0)*np.cos(lat1)*np.cos(lat2) \
+        + np.sin(lat1)*np.sin(lat2)
+    # Angle is arccos of dot product. Avoid errors caused
+    # by numerics:
+    d[d>1.0] = 1.0
+    d[d<-1.0]=-1.0
+    return 180.0/np.pi*np.arccos(d)
+
+def euclid3_to_great_circle(euclid3_distance):
+    """Convert euclidean distance between points on a unit sphere to
+    the corresponding great circle distance."""
```

Comment:
Need to specify the units of euclid3_distance and of the output.. I also don't understand the 2.0 threshold below.